tests/test_scvi.py

```diff
@@ -30,7 +30,7 @@ def test_retina():
 
 
 def test_cbmc():
-    run_benchmarks("cbmc", n_epochs=1, show_batch_mixing=False, save_path='tests/data/')
+    run_benchmarks("cite_seq_cbmc", n_epochs=1, show_batch_mixing=False, save_path='tests/data/')
```

Comment:
Is `cite_seq_cbmc` the same data as `cbmc` once it's loaded? Can we just continue to call it `cbmc` then?

Reply:
I thought Romain preprocessed the cite seq data that we're downloading now. @Edouard360 @imyiningliu Is this data just as good? If so, I'd rather use it since it's from an official source.

Reply:
I think `cbmc` files are downloaded directly from the source.